_inc/client/components/jetpack-dialogue/style.scss

```diff
@@ -61,3 +75,7 @@
 	top: rem( 100px );
 	opacity: .90;
 }
+
+.jp-dialogue__cta-container {
+	padding: rem( 8px ) 0 0 0;
```

Comment:
One too many `0` here. We can make this shorter by writing `padding: rem( 8px ) 0 0;`